packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -400,12 +506,47 @@ export class UserPool extends Resource implements IUserPool {
       }
     }
 
+    const emailVerificationSubject = props.userVerification?.emailSubject ?? 'Verify your new account';
```

Comment:
Since the cloudformation construct doesn't require email verification subject/message or sms message, should we default to empty to allow whatever cognito's defaults are to take over?

Reply:
That would be ideal, yes. 

However, at the moment, I'm unable to set up a user pool to actually test this. I'm reached out to the Cognito team to help me out here, and if the defaults there are reasonable, I'll switch to that.

Would you like me to hold this PR until then, or is it ok if I merge now and fix up later?

Reply:
That's good enough for me.